app/bundles/LeadBundle/Controller/LeadController.php

```diff
@@ -211,6 +212,10 @@ public function indexAction(
         \assert($leadDNCModel instanceof DoNotContactModel);
         $dncRepository = $leadDNCModel->getDncRepo();
 
+        /** @var UserModel $userModel */
+        $userModel            = $this->getModel('user');
```

Comment:
Add `UserModel  user$userModel` as an explicit dependency to the `indexAction` method.